hack/jenkins/release_build_and_upload.sh

```diff
@@ -136,5 +127,19 @@ do
   docker rmi -f ${IMAGE_NAME}
 done
 
+
+# upload to google bucket
+gsutil -m cp out/* "gs://$BUCKET/releases/$TAGNAME/"
+
+# Update "latest" release for non-beta/non-alpha builds
```

Comment:
@ComradeProgrammer is this releated to this PR or is this something generally fixing ? what is exactly doing?